packages/strapi-plugin-users-permissions/config/policies/rateLimit.js

```diff
@@ -22,7 +22,7 @@ module.exports = async (ctx, next) => {
       {
         interval: 1 * 60 * 1000,
         max: 5,
-        prefixKey: `${ctx.request.url}:${ctx.request.ip}`,
+        prefixKey: `${ctx.request.path}:${ctx.request.ip}`,
```

Comment:
Can you explain why this change was needed please?

Reply:
the url contains querystring params and thus you can easily bypass the rate limiting.